### ISSUE TITLE
Bundled OSX application version is reported as "Created by Qt/QMake"

### ISSUE THREAD

**Issue by m4c0** [LibrePCB/LibrePCB#655]
##### VERSION / OS / ENVIRONMENT

```
LibrePCB Version: 0.1.3
Git Revision:     56bc60d347ff67df0fe1d57807d03f0606de557f
Build Date:       2019-11-30T13:05:08
Qt Version:       5.13.2 (built against 5.13.2)
CPU Architecture: x86_64
Operating System: macOS Mojave (10.14)
Platform Plugin:  cocoa
```

##### SUMMARY

Mac OSX has a feature to "Get Info" about a file - this includes app bundles. For such bundles, it is meant to show, amongst others, the version of the app. For LibrePCB, it currently shows "Created by Qt/QMake"

##### STEPS TO REPRODUCE

1. Open the folder containing the `librepcb.app` in Finder
2. Ctrl-Click (or Right-Click) the app icon
3. Select "Get Info" 

##### EXPECTED RESULTS

The "librepcb Info" box appears, and it reports the application version (0.1.3)

##### ACTUAL RESULTS

The "librepcb Info" box appears, and it reports the application version as "Created by Qt/QMake"

##### ADDITIONAL INFORMATION

From a Mac developer perspective, this is trivial: inside the `app` bundle, there is a file named `Info.plist` inside a folder named `Contents`. That is an XML file describing the bundle. Inside that XML, there is a key-value structure which contains this snippet:

```
<key>CFBundleGetInfoString</key>
<string>Created by Qt/QMake</string>
```

I presume Qt requires some special flag somewhere in its `pro` files to properly mark the version of the app.

**Comment by spookydd** [comment]
Hello all,
I just updated to 0.1.5, on OSX (high sierra), and although I thought this had been fixed several months ago, the Get-Info version is reported as "--". Slightly different from earlier, but still not reporting the version.

**Comment by dbrgn** [comment]
How did you install LibrePCB, via the installer?

**Comment by spookydd** [comment]
Yes, the installer in the dmg.

**Comment by dbrgn** [comment]
Ok, in that case someone would need to figure out how that file can be customized when using the [Qt Installer Framework](https://doc.qt.io/qtinstallerframework/index.html).

**Comment by dbrgn** [comment]
I think we should wait with this issue until #798 is done. Not sure if that's ready by 0.1.6.

**Comment by dbrgn** [comment]
Will get fixed by #798.
